#goal definition
float32[] joints
string planning_group
---
#result definition
string estado
---
#feedback
int16 porcentaje_trayectoria